src/app/core/user/user.service.ts

```diff
@@ -78,7 +78,7 @@ export class UserService {
   private ONE_MINUTE = 60 * 1000
   private FIVE_MINUTES = 5 * 60 * 1000
   private interval$: BehaviorSubject<number> = new BehaviorSubject<number>(
-    this.ONE_MINUTE
+    10000
```

Comment:
@leomendoza123 Could you create a variable as it was before so it'd be more readable